src/styles/mixins.scss

```diff
@@ -1,69 +1,2 @@
-@import 'bulma/sass/utilities/_all.sass';
-@import 'bulma/sass/utilities/mixins.sass';
-
-@mixin lg-down {
-  // 1219.98 - 1024
-  @media only screen and (max-width: $breakpoint-lg-down) {
-    @content;
-  }
-}
-
-@mixin md-down {
-  // 1023.98 - 768
-  @media only screen and (max-width: $breakpoint-md-down) {
-    @content;
-  }
-}
-
-@mixin sm-down {
-  // 767.98 - 576
-  @media only screen and (max-width: $breakpoint-sm-down) {
-    @content;
-  }
-}
-
-@mixin xs-down {
-  // 575.98 - 414
-  @media only screen and (max-width: $breakpoint-xs-down) {
-    @content;
-  }
-}
-
-@mixin xxs-down {
-  // 413.98px - 320
-  @media only screen and (max-width: $breakpoint-xxs-down) {
-    @content;
-  }
-}
-
-@mixin section-padding {
-  // constant() is for older versions of Safari, env() all other cases
```

Comment:
Let's leave this comment in place (37 line of 'global.scss')?

Reply:
Sure, a good idea